.rubocop.yml

```diff
@@ -49,6 +49,9 @@ RSpec/MultipleExpectations:
 RSpec/ContextWording:
   Enabled: false
 
+RSpec/MessageSpies:
+  Enabled: false
```

Comment:
I had to do this so I can do this:
```ruby
expect(SchoolOrderStateAndCapUpdateService).to receive(:new).with(school: school, order_state: school.order_state, std_device_cap: 100, coms_device_cap: 0).and_call_original
```
Unless someone has any better ideas?

Reply:
I'd prefer not to disable the Rubocops if possible, they do usually indicate that something could be better. 
In this case I think it's an expectation that doesn't need to be in the spec - see my [comment on the expectation](https://github.com/DFE-Digital/get-help-with-tech/pull/782/files#r519729286) and there's a batch of suggested changes that address it